pkg/utils/validation/cidr/disjoint_test.go

```diff
@@ -258,7 +258,31 @@ var _ = Describe("utils", func() {
 			var (
 				podsCIDR     = "10.242.128.0/17"
 				servicesCIDR = "10.242.0.0/17"
-				nodesCIDR    = "10.241.0.0/16"
+				nodesCIDR    = "10.241.0.0/17"
+			)
+
+			errorList := ValidateNetworkDisjointedness(
+				field.NewPath(""),
+				&nodesCIDR,
+				&podsCIDR,
+				&servicesCIDR,
+				&seedNodesCIDR,
+				seedPodsCIDR,
+				seedServicesCIDR,
+				false,
+			)
+
+			Expect(errorList).To(ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
+				"Type":  Equal(field.ErrorTypeInvalid),
+				"Field": Equal("[].nodes"),
+			}))))
```

Comment:
Can you also include the "Detail" of the error?

Reply:
Addressed with https://github.com/gardener/gardener/pull/8353/commits/2b7c4df95fdd01469ddcccddd8e9cb17cb387eea.

However, no other test in this file actually checks `Detail`. I only adapted the new tests for now.

Reply:
Yes, I agree. But we want to follow this for the newer tests at least. There were cases that only the Error type was compared, but the actual error was something else, and the tests still passed.